### PR TITLE
DoneHW_2.2_Beheka_Misha

### PR DESCRIPTION
**Pull request by MishaBeheka**

### CODE REVIEW

File: src/main/java/core/basesyntax/MatrixOperation.java [hunk 1]
```diff
@@ -6,18 +6,51 @@
  */
 public class MatrixOperation {
     public int[] getDiagonal(int[][] matrix) {
-        return null;
+        int[] arr = new int[matrix.length];
+        int idxArr = 0;
+        for (int i = 0; i < matrix.length; i++) {
+            for (int j = 0; j < matrix.length; j++) {
```
Comment (Tarabarnik) [inline src/main/java/core/basesyntax/MatrixOperation.java]: Should we use two `for` loops here?

Comment (MishaBeheka) [reply src/main/java/core/basesyntax/MatrixOperation.java]: Thanks!

File: src/main/java/core/basesyntax/MatrixOperation.java [hunk 2]
```diff
@@ -6,18 +6,47 @@
  */
 public class MatrixOperation {
     public int[] getDiagonal(int[][] matrix) {
-        return null;
+        int[] arr = new int[matrix.length];
+        int idxArr = 0;
```
Comment (Tarabarnik) [inline src/main/java/core/basesyntax/MatrixOperation.java]: Use `i` instead of this, so we don't create redundant variables

Comment (MishaBeheka) [reply src/main/java/core/basesyntax/MatrixOperation.java]: Done

**Review by Tarabarnik — APPROVED** [review]

**Pull request closed without merging**